### DISCUSSION TITLE
Webview_MainActivity.xml

### DISCUSSION THREAD

**Discussion by Nandani321** [phaticusthiccy/WhatsAsenaDuplicated#233] [Show and tell]
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.webkit.WebView
import android.webkit.WebViewClient

class MainActivity : AppCompatActivity() {
   lateinit var mWebView : WebView
 
   override fun onCreate(savedInstanceState:Bundle?) {
     super.onCreate(savedInstanceState)
     setContentView(R.layout.activity_main)

     mWebView = findViewById(R.id.webView)

     mWebView.loadUrl("https://www.google.com")

     val webSetting = mWebView.Settings

     webSetting.javaScriptEnabled = true

     mWebView.webViewClient = WebViewClient()

}

override fun onBackPressed() {
  if(mWebView.canGoBack())
  {
    mWebView.goBack()
  }
  else
  {
  super.onBackPressed()
  }
 }
}

**Comment by phaticusthiccy** [comment]
Dude what's the deal¿
